lib/ardb/has_slug.rb

```diff
@@ -53,11 +53,12 @@ def reset_slug
 
       def ardb_has_slug_generate_slug
         slug_attr_value = self.send(self.class.ardb_has_slug_config[:attribute])
-        if !slug_attr_value || slug_attr_value.to_s.empty?
-          slug_attr_value = self.instance_eval(&self.class.ardb_has_slug_config[:source_proc])
+        slug_source     = slug_attr_value
```

Comment:
@jcredding nevermind on this comment - I better understand the intent and agree that it is a good name now.  See the comment below this one in the diff for more on helping make the intent more clear.